enterprise/server/remote_execution/snaploader/snaploader.go

```diff
@@ -760,45 +763,45 @@ func (l *FileCacheLoader) cacheActionResult(ctx context.Context, key *fcpb.Snaps
 			}
 			log.CtxInfof(ctx, "Cached remote snapshot manifest %s", snapshotDebugString(ctx, l.env, snapshotSpecificKey, true /*remote*/, snapshotID))
 		}
-
-		return nil
```

Comment:
It's hard to tell from this diff, but I just removed this `return nil`  and added the `if *snaputil.EnableLocalSnapshotSharing && opts.CacheManifestLocally` below. The rest of the logic is unchanged